styles.js

```diff
@@ -0,0 +1 @@
+require('./src/style.scss');
```

Comment:
with `import` it didn't work... I didn't care enough to investigate so I left it with `require` :)